pages/en/learn/command-line/run-nodejs-scripts-from-the-command-line.md

```diff
@@ -69,3 +69,5 @@ Run the application using the `nodemon` command followed by the application's fi
 ```bash
 nodemon app.js
 ```
+
+> Note: nodejs 16 and above has an **exerimental** watch mode. You can use `node --watch app.js` to run the application in watch mode. You can read about this in the [api docs](https://nodejs.org/docs/latest-v18.x/api/cli.html#--watch).
```

Comment:
Also here, we should have a section for watch mode instead of redirecting to API docs.

Reply:
for moment it's experimental so I don't thinks it's good idea to have big section. WDYT ?

Reply:
It's becoming stable pretty soon, anyways. I don't see why including it. But if it's experimental and the concern is stability, then why mentioning it on Learn at all?

Reply:
Yes, we could include it. 

> But if it's experimental and the concern is stability, then why mentioning it on Learn at all?

in my opinion it could be mentioned so that users can test it and give their feedback

Reply:
Right, but this is a learn section. I feel that if we add something we should teach them. Our API docs are known for not being learn-friendly.

Reply:
I'm re-wrintting se § "Restart the application automatically"